utils/freifunk-berlin-migration/uci-defaults/999_freifunk-berlin-migration.sh

```diff
@@ -79,10 +81,11 @@ migrate () {
     update_dhcp_lease_config
     update_wireless_ht20_config
     update_luci_statistics_config
-    update_crontab_memory_leak_fix
     update_olsr_smart_gateway_threshold
   fi
 
+  update_collectd_memory_leak_hotfix
+
```

Comment:
Why don't you create a new `if semverLT ${OLD_VERSION} "0.2.0"; then`-block and put `update_collectd_memory_leak_hotfix` in there?